src/core/iomgr/pollset_windows.h

```diff
@@ -42,10 +42,17 @@
    nature of the IO completion ports. A Windows "pollset" is merely a mutex
    and a condition variable, used to synchronize with the IOCP. */
```

Comment:
So the above comment should be updated to describe the new version of the Windows pollset, something like:

A Windows "pollset" is merely a mutex used to synchronize with the IOCP, along with a queue of workers that each has its own condition variable.